### PR TITLE
Privacy policy - 0x

### PR DESCRIPTION
**Pull request by 409H**
* Add mention about third-party, proxy, and 0x

### CODE REVIEW

File: src/assets/content/privacy-policy.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Some examples of data we **do not** collect:
 
 We do not utilize any tracking cookies. Any individualized (though anonymized) records collected by Matomo (as described above) are automatically purged after one year. We may retain anonymized, aggregated reports perpetually as part of our ongoing business operations and optimization.
 
+Some of our non-proxied third-party providers (such as [The Graph](https://thegraph.com/) and [CoinGecko](https://www.coingecko.com/)) will have access to your IP address, various asset tickers, and ethereum address in the request to their APIs. You can find their Terms of Service and Privacy Policy near the bottom of this document. Generally we proxy requests to third-party providers, which would mask your originating IP to the end third-party provider as our API server IP - we do not store these requests.
```
Comment (LuckyLuker) [inline src/assets/content/privacy-policy.md:73]: and ethereum address ->  and Ethereum address

Comment (409H) [reply src/assets/content/privacy-policy.md]: amended in 17bbed1

**Review by GuillaumeRx — DISMISSED** [review]
lgtm

**Comment by GuillaumeRx** [comment]
Please rebase your branch when you have time 👍

**Review by GuillaumeRx — APPROVED** [review]

**Pull request merged**